### ISSUE TITLE
Add MJS as an extension for mimetype.rs

### ISSUE THREAD

**Issue by Deviouslrd** [tauri-apps/wry#250]
**Describe the bug**
```
Failed to load module script: The server responded with a non-JavaScript MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

**Steps To Reproduce**
Import that gave me the issue:
```
<script type="module" src="./functions/main_links.mjs"></script>
```

**Expected behavior**
The file to be loaded correctly, like the other filetypes included in the mimetype file.

**Would you assign yourself to resolve this bug?**
- [ ] Yes
- [x] No

**Additional context**
Could have done a PR for it, but decided against it, so im just doing an issue instead.

**Comment by wusyong** [comment]
It seems [Common MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) from MDN states that `.mjs` extension should belong to `text/javascript`.
You probably want to change to `<script type="text/javascript" src="./functions/main_links.mjs"></script>`

**Comment by lucasfernog** [comment]
I think the problem might be the extension not being matched on mimetype.rs

**Comment by wusyong** [comment]
yes I tested with custom protocol example and change the tyep to `type="module"`. It will result in the same error too.

**Comment by Deviouslrd** [comment]
Should also mention it needs to be `type="module"` in my case because the file contains import statements to require various things. (one namely to get the package.json version)

**Comment by wusyong** [comment]
Closed by #296. Now it's up to users to decide what mime type is.
